tools/mason/MasonPublish.chpl

```diff
@@ -38,52 +38,28 @@ proc masonPublish(args: [] string) throws {
       masonPublishHelp();
       exit(0);
     }
-    if args.size == 4 {
-      masonPublishHelp();
-      exit(0);
-    }
     var dry = false;
+    var path = '';
     var username = getUsername();
-    if username == '' {
-      throw new owned MasonError('Must have remote origin set up in repository to publish');
+    var isLocal = false;
+
+    for arg in args[2..] {
+      if arg == '--dry-run' then dry = true;
+      else path = arg;
```

Comment:
Are we catching the case if multiple registries are given, e.g. `mason publish foo bar baz` ?